site/content/docs/latest/tutorials/kubeapps-on-tkg/step-3.md

```diff
@@ -5,60 +5,65 @@ Once Kubeapps is up and running on the VMware Tanzu™ Kubernetes Grid™ cluste
+- `Repository Authorization` field if selected `Docker Registry` options, needs to specifies an `imagePullSecret` for images that are to be pulled from the private registry. It is possible to create a fresh secret or choose an existing one.
 
-  ![Package repositories modal part 2](../../img/kubeapps-on-tkg/apprepository-detail-types.png)
+  ![Package repositories modal part 3](../../img/kubeapps-on-tkg/apprepository-authentication.png)
 
-- The _Associate Docker Registry_ field specifies an `imagePullSecret` for images that are to be pulled from the private registry. It is possible to create a fresh secret or choose an existing one.
+- For OCI registrie, it is necessary to also manually specify the list of artifacts to fetch in the **List of Repositories** field (as there is no index yet). Additionally, artifacts can be excluded using regular expressions if required.
 
-  ![Package repositories modal part 3](../../img/kubeapps-on-tkg/apprepository-detail-credentials.png)
+  ![Package repositories modal part 3](../../img/kubeapps-on-tkg/apprepository-filtering.png)
 
-- The _Custom CA Certificate_ field specifies a CA certificate to use (with an option to skip the TLS verification if required).
-- The _Custom Sync Job_ field specifies the synchronization template to use when periodically pulling the latest changes from the application repository.
+- `Synchronization Interval` field specifies the synchronization template to use when periodically pulling the latest changes from the application repository (available for Helm via Flux and Carvel).
+- `Custom CA Certificate` field specifies a CA certificate to use (with an option to skip the TLS verification if required).
 
-  ![Package repositories modal part 4](../../img/kubeapps-on-tkg/apprepository-detail-advanced.png)
+  ![Package repositories modal part 3](../../img/kubeapps-on-tkg/apprepository-advanced.png)
 
-6. Click the _Install Repo_ button to finish the process and add the repository to Kubeapps.
+6. Click the **Install Repository** button to finish the process and add the repository to Kubeapps.
 
-Retrieving the catalog from the repository will take a few minutes. Once complete, the contents of the repository will become available in Kubeapps. To check the progress of the synchronization task, use the following command, replacing REPOSITORY-NAME with the name configured above:
+Retrieving the catalog from the repository takes a few minutes. Once complete, the contents of the repository become available in Kubeapps. To check the progress of the synchronization task, use the following command, replacing `REPOSITORY-NAME` with the name configured above:
```

Comment:
This is just for the Helm plugin, I'd rather remove this part:
> To check the progress of the synchronization task, use the following command, replacing `REPOSITORY-NAME` with the name configured above

Reply:
Thanks for the revision. Updated!